examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -0,0 +1,58 @@
+"""
+Demonstrate basic surface plotting with plot_surf_stat_map
+==========================================================
```

Comment:
Or just upload them to the example dataset just as we did for the fsaverage mesh?